### PR TITLE
Add French language

### PR DESCRIPTION
**Pull request by Ilithy**
I am not a coder, so I may have made mistakes, do not hesitate to point them out to me, I will correct them

Thank you very much for this very beautiful launcher

### CODE REVIEW

**Comment by nivisi** [comment]
@Ilithy can you please, instead of forking, download the repo and create a branch for this PR? So I will be able to test it on my machine and see why the analyzer failed.

**Comment by Ilithy** [comment]
I'll try, I cancel this PR, and do what you ask

**Comment by Ilithy** [comment]
@nivisi I created a different branch on my Fork, but I can't create a branch on your repo, is that ok without PR? should I do a PR under other branches, for example the master branch? Or just provide you the files?
I am at your disposal, do not hesitate, my apologies for the complications.

**Comment by Ilithy** [comment]
[LeafyLauncher-Ilithy-French-patch-I.zip](https://github.com/nivisi/LeafyLauncher/files/8239234/LeafyLauncher-Ilithy-French-patch-I.zip)

here are the files of the FR patch branch,
I don't dare to do anything else without your prior agreement.

**Comment by nivisi** [comment]
@Ilithy thanks for your great effort, it's truly highly appreciated 🙂 I've written [some contribution guidelines notes](https://github.com/nivisi/LeafyLauncher/wiki/Contribution-Guidelines), you can check them out. They're not great yet, I will work on them in the future.

TL;DR, for this particular case: 
- Create a branch called `feature/add-french-language`;
- Open a PR with title `Add French language`;
- Put `Resolves #203` to the top of the PR description.

Try it out in the fork and open a PR like you did previously. It should work.

**Comment by nivisi** [comment]
@Ilithy let me know if something is unclear I'm happy to help.

**Comment by Ilithy** [comment]
@nivisi Thank you very much for your help, your guideline is very well written, and also very practical, bravo. I have done this as requested (hopefully) on the PR #206
